tests/testthat/test-guides.R

```diff
@@ -944,8 +951,8 @@ test_that("guides title and text are positioned correctly", {
     geom_point() +
     # setting the order explicitly removes the risk for failed doppelgangers
     # due to legends switching order
-    guides(shape = guide_legend(order = 1),
-           color = guide_colorbar(order = 2)) +
+    guides(size  = guide_legend(order = 2),
+           color = guide_colorbar(order = 1)) +
```

Comment:
This plot wasn't using the `shape` aesthetic, so the test was wrong. The checks in this PR correctly picked up on this.